wiki/Beatmapping/Hitsound/id.md

```diff
@@ -0,0 +1,36 @@
+---
+stub: true
+tags:
+  - hit sounds
+  - hitsounds
+---
+
+# Hitsound
+
+*Lihat juga: [Menambah Kustom hitsounds](/wiki/Guides/Adding_Custom_Hitsounds)*
+
+**Hitsounds** adalah efek suara saat bermain osu! ketika pengguna berinteraksi dengan [hit objects](/wiki/Hit_Objects). Hitsound biasanya digunakan untuk memberikan umpan balik suara kepada pengguna untuk membantu mereka dalam menilai [accuracy](/wiki/Accuracy/) yang terkait dengan lagu tersebut.
+
+Hitsound terdiri dari sampel default yang disebut "hit normal", dan kombinasi sampel whistle, finish, atau clap. Setiap sampel memiliki satu dari tiga "sampel set" yang mengubah gayanya: `Normal`,` Soft`, atau `Drum`.
+
+Beatmapper dapat mengganti secara default sampel suara dengan kustom sampel folder [beatmapset](/wiki/Beatmaps/Beatmapsets). [Skin](/wiki/Skinning) pengguna juga bisa mengganti hitsound default untuk seluruh beatmap. Detail untuk sampel hitsound dapat ditemukan didalam artikel [hitsound skinning](/wiki/Skinnning/Sounds#hitsounds).
```

Comment:
```suggestion
Beatmapper dapat mengganti secara default sampel suara dengan kustom sampel di dalam folder [beatmapset](/wiki/Beatmaps/Beatmapsets). [Skin](/wiki/Skinning) pemain juga bisa mengganti hitsound default untuk seluruh beatmap. Detail untuk sampel hitsound dapat ditemukan didalam artikel [hitsound skinning](/wiki/Skinnning/Sounds#hitsounds).
```
Needs clarity as currently it could be seen as specific folder inside a beatmapset flder, while in reality its the beatmapset's folder